app/views/admin/users/voters.html.haml

```diff
@@ -29,8 +29,9 @@ Search: &nbsp;
         %td{style: 'max-width: 250px;'}
```

Comment:
I think we might actually want to leave this showing the fake numbers. It's an admin-only page, where it might be useful to make it obvious who has real numbers and who doesn't

Reply:
just this one--it's for admins on the DtV side, where we want to see how the sausage is made. Other views are customer-facing, where it would just confuse.

Reply:
OK great. I updated the branch f you want to review.